src/science/canopy/cable_canopy.F90

```diff
@@ -668,6 +668,10 @@ SUBROUTINE define_canopy(bal,rad,rough,air,met,dels,ssnow,soil,veg, canopy,clima
 
        ENDDO
 
+       ! INH #335 - we don't need to weight components of %epot by %transd
+       ! however coupled model uses %wetfac_cs so overwrite here before testing in ACCESS
+       canopy%epot = (canopy%fevw_pot + ssnow%potev/ssnow%cls) * dels/air%rlam
```

Comment:
Ideally we would change `%epot` and hence `%wetfac_cs` in the section above this change - and use the correct value in both offline and coupled.  However since `%wetfac_cs` is one of the coupling variables - and one that I think does get used - I think this needs independent testing within ACCESS (and so is a separate issue).

In the interim I opted to adjust `%epot` after the evaluation of `%wetfac_cs` which will allow updates to CABLE3 to be ported across to ACCESS as a first step towards resyncing the code bases.